editor/src/clj/editor/pose.clj

```diff
@@ -223,10 +223,10 @@
     (cond-> child-pose
 
             parent-scaled
-            (assoc :scale (multiply-vector child-scale parent-scale))
+            (assoc :scale (multiply-vector parent-scale child-scale))
```

Comment:
Order doesn't matter when multiplying vectors, but it looks better as the function is called `pre-multiply` and it also corresponds to argument order of the `multiply-quaternion` call (where the order *does* matter!) below.